### DISCUSSION TITLE
Does OpenDJ support the NetBIOS domainname\username format

### DISCUSSION THREAD

**Discussion by tedxu-au** [OpenIdentityPlatform/OpenDJ#338] [Q&A] [answered]
Hi All. I'm currently looking for an LDAP server which support authenticating via the NetBIOS domainname\username format. I've also looked into openldap, but unfortunately it seems that openldap does not support this.

Does OpenDJ support, or can be configured to support the NetBIOS / pre-Windows 2000 domain name format? Thanks.

**Comment by maximthomas** [comment, marked as answer]
Hi, unfortunately, OpenDJ supports only DN binding in its native API. As an alternative, I would suggest implementing searching user DN by a user alias attribute and then authenticating the user using his DN in your software

**Comment by vharseko** [comment]
...PLAIN SASL authentication (with a user name and password), SASL GSSAPI authentication (Kerberos V5), SASL CRAM MD5, and DIGEST MD5 authentication. see [Regular Expression identity mapper](https://doc.openidentityplatform.org/opendj/admin-guide/#client-auth)
